### ISSUE TITLE
Accessibility: Buttons are not operable by keyboard

### ISSUE THREAD

**Issue by nedredmond** [inklewriter/freeinklewriter#74]
Every button on the site is a div, making them invisible to the browser's accessibility API. They cannot be discovered as interactive elements by screen readers, can't be navigated or activated by keyboard.

## Suggested fix:
Buttons should be implemented with the `button` tag.
This will require more HTML to reset styles, but [a shortcut ](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/all)would be starting the class styles with `all: unset`.

## ADA Standards Violations:
- https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html
- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value
